tsconfig.json

```diff
@@ -8,7 +8,12 @@
     "inlineSourceMap": true,
     "declaration": true,
     "newLine": "LF",
-    "outDir": "dist"
+    "outDir": "dist",
+    "lib": [
+      "dom",
```

Comment:
Necessary because we're no longer using the `Simple` namespace for things like `HTMLElement` and there is no default `lib` array for the `es2017` target.